modules/multiplayer/editor/editor_network_profiler.cpp

```diff
@@ -42,6 +43,7 @@ void EditorNetworkProfiler::_bind_methods() {
 
 void EditorNetworkProfiler::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
```

Comment:
This shouldn't be necessary, `THEME_CHANGED` always happens together with `ENTER_TREE`.